example/res/layout/affirm_payment_activity.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
might be worth consolidating this with the `klarna_payment_activity.xml` looks like they're both the same minus the content of the strings. Could just set the value of the text in `onCreate()`. Then in the future all we need to do is set the values of the text boxes and we don't need to add more XML.

Bonus if you did this in jetpack compose rather than XML but I don't expect you to.